bigframes/core/compile/ibis_compiler/scalar_op_compiler.py

```diff
@@ -278,3 +283,18 @@ def isnanornull(arg):
 @scalar_op_compiler.register_unary_op(numeric_ops.isfinite_op)
 def isfinite(arg):
     return arg.isinf().negate() & arg.isnan().negate()
+
+
+@scalar_op_compiler.register_ternary_op(geo_ops.StRegionStatsOp, pass_op=True)
+def st_regionstats(
+    geography: ibis_types.Value,
+    raster: ibis_types.Value,
+    band: ibis_types.Value,
+    op: geo_ops.StRegionStatsOp,
+):
+    args = [geography, raster, band]
+    if op.options:
+        args.append(bigframes_vendored.ibis.literal(op.options, type="json"))
+    return bigframes_vendored.ibis.remote_function(
+        "st_regionstats", args, output_type="struct<min: float, max: float, sum: float, count: int, mean: float>"  # type: ignore
```

Comment:
Per https://cloud.google.com/bigquery/docs/reference/standard-sql/geography_functions#st_regionstats it should also include area.